RetailCoder.VBE/UI/Command/CommandBase.cs

```diff
@@ -15,24 +15,18 @@ public abstract class CommandBase : ICommand
 
         protected CommandBase(ILogger logger)
         {
-            _logger = logger;
+            Logger = logger;
```

Comment:
doesn't this require `Logger` to have a `private set;`? or possibly a `readonly`?